sdk/iot/Azure.Iot.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -1,4 +1,913 @@
+        public Azure.Iot.Hub.Service.Models.AuthenticationMechanismType? Type { get { throw null; } set { } }
+        public Azure.Iot.Hub.Service.Models.X509Thumbprint X509Thumbprint { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct AuthenticationMechanismType : System.IEquatable<Azure.Iot.Hub.Service.Models.AuthenticationMechanismType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public AuthenticationMechanismType(string value) { throw null; }
+        public static Azure.Iot.Hub.Service.Models.AuthenticationMechanismType CertificateAuthority { get { throw null; } }
+        public static Azure.Iot.Hub.Service.Models.AuthenticationMechanismType None { get { throw null; } }
+        public static Azure.Iot.Hub.Service.Models.AuthenticationMechanismType Sas { get { throw null; } }
+        public static Azure.Iot.Hub.Service.Models.AuthenticationMechanismType SelfSigned { get { throw null; } }
+        public bool Equals(Azure.Iot.Hub.Service.Models.AuthenticationMechanismType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.Iot.Hub.Service.Models.AuthenticationMechanismType left, Azure.Iot.Hub.Service.Models.AuthenticationMechanismType right) { throw null; }
+        public static implicit operator Azure.Iot.Hub.Service.Models.AuthenticationMechanismType (string value) { throw null; }
+        public static bool operator !=(Azure.Iot.Hub.Service.Models.AuthenticationMechanismType left, Azure.Iot.Hub.Service.Models.AuthenticationMechanismType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class BulkRegistryOperationResult
+    {
+        internal BulkRegistryOperationResult() { }
+        public System.Collections.Generic.IReadOnlyList<Azure.Iot.Hub.Service.Models.DeviceRegistryOperationError> Errors { get { throw null; } }
+        public bool? IsSuccessful { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.Iot.Hub.Service.Models.DeviceRegistryOperationWarning> Warnings { get { throw null; } }
+    }
+    public partial class CloudToDeviceMethodRequest
+    {
+        public CloudToDeviceMethodRequest() { }
+        public int? ConnectTimeoutInSeconds { get { throw null; } set { } }
+        public string MethodName { get { throw null; } set { } }
+        public object Payload { get { throw null; } set { } }
```

Comment:
string, not object

Reply:
actually, nevermind about this. This API doc has a lot going on that isn't related to modules specifically.